config/v1/types_scheduling.go

```diff
@@ -25,6 +25,16 @@ type SchedulingSpec struct {
 	// The namespace for this configmap is openshift-config. 
 	// +optional
 	Policy ConfigMapNameReference `json:"policy,omitempty"`
+	// defaultNodeSelector helps set the cluster-wide default node selector to 
+	// restrict pod placement to specific nodes. This is applied to the pods 
+	// created in all projects without a specified nodeSelector value.
+	// For example,
+	// defaultNodeSelector: "type=user-node,region=east" would set nodeSelector
+	// field in pod spec to "type=user-node,region=east" to all pods created
+	// in all projects.Projects having project-wide node selectors won't be 
```

Comment:
"project-wide"?  is this an annotation on the namespace, if so, indicate what it is please.